x-pack/legacy/plugins/siem/public/pages/detection_engine/rules/components/anomaly_threshold_slider/index.tsx

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { useCallback } from 'react';
+import styled from 'styled-components';
+import { EuiFlexGrid, EuiFlexItem, EuiRange, EuiFormRow } from '@elastic/eui';
+
+import { FieldHook } from '../../../../../shared_imports';
+
+interface AnomalyThresholdSliderProps {
+  field: FieldHook;
+}
+type Event = React.ChangeEvent<HTMLInputElement>;
+type EventArg = Event | React.MouseEvent<HTMLButtonElement>;
+
+const Wrapper = styled(EuiFormRow)``;
```

Comment:
Do want, need this empty wrapper? Seems like you can just use `EuiFormRow` as is?